modules/FormModel.js

```diff
@@ -322,40 +495,111 @@
-     * @method listenToComputedValuesDependency
+     * @param [field] {String} the field name that it will start listening to. If no field is given, it will listen to the general 'change' event.
+     * @param computedAlias {String} the name/alias of the computed mapping being used.
+     * @method __listenToComputedValuesDependency
+     * @private
      */
-    listenToComputedValuesDependency: function(computedConfig, model, field) {
-      var eventName = 'change:' + field;
-      this.listenTo(model, 'change:' + field, _.bind(this.__invokeComputedPull,
-          {formModel: this, models: computedConfig.models, pull: computedConfig.pull}));
+    __listenToComputedValuesDependency: function(model, field, computedAlias) {
+      var eventName = field ? ('change:' + field) : 'change';
+      this.listenTo(model, eventName, _.bind(this.__invokeComputedPull,
+          {formModel: this, alias: computedAlias}));
       this.__currentUpdateEvents.push({model: model, eventName: eventName});
     },
 
-    /************** Private methods **************/
+    /**
+     * Returns the models that a currently being tracked that are part of a computed mapping
+     * If there is a missing model (a model alias is referenced but no model instance is bound to that alias), then it will return undefined.
+     * @method __getComputedModels
+     * @param computedAlias {String} the name/alias of the computed mapping
+     * @return {Object} a map from model name/alias to model instance. If there is a missing model (an model alias is referenced but no model
+     *   instance is bound to that alias), then it will return undefined.
+     * @private
+     */
+    __getComputedModels: function(computedAlias) {
+      var hasAllModels = !_.isUndefined(this.getMapping(computedAlias)),
+        models = {};
+      _.each(this.__getModelAliases(computedAlias), function(modelAlias) {
+        var model = this.getTrackedModel(modelAlias);
+        if (model) {
+          models[modelAlias] = model;
+        } else {
+          hasAllModels = false;
+        }
+      }, this);
+      return hasAllModels ? models : undefined;
+    },
+
+    /**
+     * Returns the aliases/names of models referenced in the computed mapping with the given alias
+     * @method __getModelAliases
+     * @param computedAliasOrConfig {String or Object} the name/alias of the computed mapping or the computed mapping itself as
+     *   an object if it hasn't been added as a mapping yet.
+     * @return {Array of Strings} an array of the model names/aliases referenced inside the computed mapping
+     * @private
+     */
+    __getModelAliases: function(computedAliasOrConfig) {
+      var config,
+        modelAliases = [];
+      if (_.isString(computedAliasOrConfig)) {
+        config = this.getMapping(computedAliasOrConfig);
+      } else {
+        config = computedAliasOrConfig;
+      }
+      _.each(config.mapping, function(mapping, key) {
```

Comment:
modelAliases = _.map(config.mapping, function(mapping, key) { ... });

writing it out with chaining to see how it looks:
```
modelAliases = _.chain(config.mapping)
    .map(function(mapping, key) {
      return key;
    })
    .filter(function(key) {
      return key != 'pull' && key != 'push';
    })
    .value();
```

nope, not better to do chaining.